jangaroo-maven/jangaroo-maven-plugin/src/main/java/net/jangaroo/jooc/mvnplugin/WorkspaceConverterMojo.java

```diff
@@ -184,369 +185,374 @@ public void execute() throws MojoExecutionException, MojoFailureException {
-        Map<String, String> sortedDependencies = new TreeMap<>();
-        if (packageJson.getDependencies() != null) {
-          packageJson.getDependencies().entrySet().stream()
-                  .sorted(Map.Entry.comparingByKey())
-                  .forEach(entry -> sortedDependencies.put(entry.getKey(), entry.getValue()));
-        }
-        packageJson.setDependencies(sortedDependencies);
-        Map<String, String> sortedDevDependencies = new TreeMap<>();
-        if (packageJson.getDevDependencies() != null) {
-          packageJson.getDevDependencies().entrySet().stream()
-                  .sorted(Map.Entry.comparingByKey())
-                  .forEach(entry -> sortedDevDependencies.put(entry.getKey(), entry.getValue()));
-        }
-        packageJson.setDevDependencies(sortedDevDependencies);
+      PackageJson packageJson = new PackageJson(additionalJsonEntries);
+      packageJson.setName(aPackage.getName());
+      packageJson.setVersion(aPackage.getVersion());
+      if (mavenModule.getData().getOrganization() != null) {
+        packageJson.setAuthor(mavenModule.getData().getOrganization().getName());
+      }
+      packageJson.setLicense("MIT");
+      packageJson.setPrivat(true);
+      aPackage.getDependencies().stream().collect(Collectors.toMap(Package::getName, Package::getVersion)).forEach(packageJson::addDependency);
+      aPackage.getDevDependencies().stream().collect(Collectors.toMap(Package::getName, Package::getVersion)).forEach(packageJson::addDevDependency);
+      Map<String, String> sortedDependencies = new TreeMap<>();
+      if (packageJson.getDependencies() != null) {
+        packageJson.getDependencies().entrySet().stream()
+                .sorted(Map.Entry.comparingByKey())
+                .forEach(entry -> sortedDependencies.put(entry.getKey(), entry.getValue()));
+      }
+      packageJson.setDependencies(sortedDependencies);
+      Map<String, String> sortedDevDependencies = new TreeMap<>();
+      if (packageJson.getDevDependencies() != null) {
+        packageJson.getDevDependencies().entrySet().stream()
+                .sorted(Map.Entry.comparingByKey())
+                .forEach(entry -> sortedDevDependencies.put(entry.getKey(), entry.getValue()));
+      }
+      packageJson.setDevDependencies(sortedDevDependencies);
+      try {
         FileUtils.write(new File(targetPackageJson), objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(packageJson).concat("\n"));
+      } catch (IOException e) {
+        throw new MojoFailureException(e.getMessage(), e.getCause());
+      }
+    }
+  }
+
+  private void loadAndCopyResource(String fileName) throws IOException {
+    if (Paths.get(convertedWorkspaceTarget + fileName).toFile().exists()) {
+      InputStream npmrc = getClass().getResourceAsStream("/net/jangaroo/jooc/mvnplugin/" + fileName);
+      if (npmrc != null) {
+        Files.copy(npmrc, Paths.get(convertedWorkspaceTarget, fileName), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Introduce local var for Paths.get(...). Your second variant, using two parameters, is better than adding strings (convertedWorkspaceTarget might miss a trainling slash).